pyxem/tests/signals/test_diffraction2d.py

```diff
@@ -862,7 +862,7 @@ def test_non_uniform_chunks(self):
     def test_return_shifts_non_lazy(self):
         s = self.s
         s_shifts = s.center_direct_beam(method="blur", sigma=1, return_shifts=True)
-        assert s_shifts._lazy is False
+        assert s_shifts._lazy is True
```

Comment:
Why this change? I'm thinking a non-lazy input should return a non-lazy output?

Reply:
Yes that is probably what should happen.  I didn't look that over too well I'll make that change back.